### PR TITLE
Implement RSS

### PR DESCRIPTION
**Pull request by domingoruiz**
Following the instruction [https://gohugo.io/templates/rss/](https://gohugo.io/templates/rss/) I have implemented the RSS functionality.

Thanks

### CODE REVIEW

**Comment by FoggyMtnDrifter** [comment]
I'll have to check this out a little later, Hugo generates RSS files by default without the need of a template, so this may not even be needed.

File: layouts/_default/rss.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+{{- $limit := .Site.Config.Services.RSS.Limit -}}
+{{- if ge $limit 1 -}}
+{{- $pages = $pages | first $limit -}}
+{{- end -}}
+{{- printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\"?>" | safeHTML }}
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+  <channel>
+    <title>{{ if eq  .Title  .Site.Title }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{.}} on {{ end }}{{ .Site.Title }}{{ end }}</title>
+    <link>{{ .Permalink }}</link>
+    <description>RRS channel {{ if ne  .Title  .Site.Title }}{{ with .Title }}in {{.}} {{ end }}{{ end }}on {{ .Site.Title }}</description>
```
Comment (fabaff) [inline layouts/_default/rss.xml]:
```suggestion
    <description>RSS channel {{ if ne  .Title  .Site.Title }}{{ with .Title }}in {{.}} {{ end }}{{ end }}on {{ .Site.Title }}</description>
```

**Review by fabaff — CHANGES_REQUESTED** [review]

**Comment by domingoruiz** [comment]
I know, in fact that is Hugo's own template, I got it from his Github. The only thing that instead of using the default template I have modified it a bit, deleting Hugo's references and customizing some errors that I have seen in the expressions in English.

File: layouts/_default/rss.xml (hunk 1 above)
Comment (domingoruiz) [reply layouts/_default/rss.xml]: Ok very good. Thanks.

**Pull request merged**